apps/angular/src/app/pages/main-page.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="controls">
+	<camp-anime-type-filter
+		[types]="(animeTypeFilter$ | async) ?? []"
+		(filterValueEmitter)="onFilterValueChange($event)"
+	/>
+	<camp-search-form
+		[search]="(search$ | async) ?? ''"
+		(searchValueEmitter)="onSearchValueChange($event)"
+	/>
+</div>
+@if (hasFetchingError$ | async) {
+	<h2 class="error">Fetching error occurred!</h2>
+} @else {
+	<camp-anime-table
+		[animeList]="(animeList$ | async) ?? []"
+		(sortValueEmitter)="onSortChange($event)"
+	/>
+}
+<camp-paginator
+	[pageSizes]="pageSizes"
+	[pageSize]="(pageSize$ | async) ?? 25"
+	[pageNumber]="(pageNumber$ | async) ?? 0"
+	[totalCount]="totalCount"
+	(pageEventEmitter)="onPageEvent($event)"
+/>
```

Comment:
It's a bit strange that we display pagination even if table is not visible. Let's adjust the layout:
```suggestion
@if (hasFetchingError$ | async) {
	<h2 class="error">Fetching error occurred!</h2>
} @else {
	<camp-anime-table
		[animeList]="(animeList$ | async) ?? []"
		(sortValueEmitter)="onSortChange($event)"
	/>
	
	<camp-paginator
		[pageSizes]="pageSizes"
		[pageSize]="(pageSize$ | async) ?? 25"
		[pageNumber]="(pageNumber$ | async) ?? 0"
		[totalCount]="totalCount"
		(pageEventEmitter)="onPageEvent($event)"
	/>
}

```